Target.java

```diff
@@ -1,17 +1,42 @@
-/*
- * File: Target.java
- * Name: 
- * Section Leader: 
- * -----------------
- * This file is the starter file for the Target problem.
- */
-
-import acm.graphics.*;
-import acm.program.*;
-import java.awt.*;
-
-public class Target extends GraphicsProgram {	
-	public void run() {
-		/* You fill this in. */
-	}
-}
+/*
+ * File: Target.java
+ * Name: 
+ * Section Leader: 
+ * -----------------
+ * This file is the starter file for the Target problem.
+ */
+
+import acm.graphics.*;
+import acm.program.*;
+import java.awt.*;
+
+public class Target extends GraphicsProgram {	
+	public void run() {
+		/*
+		 * This program draws an archery target
+		 */

```

Comment:
These values should have been declared as constants outside of the run method. So something like:
 
```
private static final int INCH_TO_PIX = 72;
private static final double MAXI_RADIUS = 1 * INCH_TO_PIX;
private static final double MEDI_RADIUS = 0.65 * INCH_TO_PIX;
private static final double MINI_RADIUS = 0.3 * INCH_TO_PIX;
```